sdk/go/internal/querybuilder/marshal.go

```diff
@@ -137,6 +139,10 @@ func marshalCustom(ctx context.Context, v reflect.Value) (string, error) {
 }
 
 func IsZeroValue(value any) bool {
+	if value == nil {
```

Comment:
Since nil doesn't have a type assigned, this needs to be handled in a special way